pkgs/development/python-modules/tomlkit/default.nix

```diff
@@ -10,11 +10,11 @@
 
 buildPythonPackage rec {
   pname = "tomlkit";
-  version = "0.11.6";
+  version = "0.11.4";
```

Comment:
Downgrading packages should be exceptional: Is the new behavior a bug or a feature?
- If it's a feature we should either
  - Introduce a new `tomlkit_11_4` attribute and use that, or
  - Define the older version internally to the `pkgs.formats` code (this way we don't have a new attribute to maintain)  
- If it's a bug we should add a comment here saying that we're waiting for the bug to get fixed before upgrading.

Reply:
You're absolutely right :)

It turns out that tomlkit 0.11.8 (the latest version) doesn't have this bug either. So doing an upgrade and introducing the tests seems way more appropriate.

The challenge is that tomlkit is now built with poetry and.... poetry also depends on tomlkit.

So I think until we bootstrap poetry itself, we need to keep 0.11.6 as `tomlkit_11_6`, similar to your suggestion.

I'll make those changes. Hopefully I don't mess it up :-)

Thanks for the quick review, btw!